make/mt-config/mt-presets.mk

```diff
@@ -99,19 +99,19 @@ else ifeq ($(PRESET), CC3200_TLS_SOCKET)
     XI_TARGET_PLATFORM = cc3200
 
 # -------------------------------------------------------
-# ST Micro STM32F4
-else ifeq ($(PRESET), STM32F4)
-    CONFIG = $(CONFIG_STM32F4)
+# ST Micro STM32FX
```

Comment:
I'm a bit confused. There's a Generic STM32FX build configuration, and a more specific STM32FX Nucleo Wifi configuration which is very specific.

When would the STM32FX build configuration be applicable?  Or is this just a generic work in progress for our next STM32F platform?